Dockerfile

```diff
@@ -16,23 +18,40 @@ RUN dotnet restore NineChronicles.Headless.Executable
 
 # Copy everything else and build
 COPY . ./
-RUN dotnet publish NineChronicles.Headless.Executable/NineChronicles.Headless.Executable.csproj \
+RUN <<EOF
+#!/bin/bash
+echo "TARGETPLATFROM=$TARGETPLATFORM"
+if [[ "$TARGETPLATFORM" = "linux/amd64" ]]
+then
+  dotnet publish NineChronicles.Headless.Executable/NineChronicles.Headless.Executable.csproj \
     -c Release \
     -r linux-x64 \
     -o out \
     --self-contained \
     --version-suffix $COMMIT
+elif [[ "$TARGETPLATFORM" = "linux/arm64" ]]
+then
+  dotnet publish NineChronicles.Headless.Executable/NineChronicles.Headless.Executable.csproj \
+    -c Release \
+    -r linux-arm64 \
+    -o out \
+    --self-contained \
+    --version-suffix $COMMIT
+else
+  echo "Not supported target platform: '$TARGETPLATFORM'."
+  exit -1
+fi
+EOF
 
 # Build runtime image
-FROM mcr.microsoft.com/dotnet/aspnet:8.0
+FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/aspnet:8.0-bookworm-slim
```

Comment:
Note: according to the [dotnet/dotnet-docker repo](https://github.com/dotnet/dotnet-docker/blob/main/README.aspnet.md#linux-amd64-tags), `8.0-bookworm-slim` is same with the `8.0`